testing/tests/DevExpress.ui.widgets/toolbar.tests.js

```diff
@@ -1715,3 +1715,133 @@ QUnit.test("T430159 dropdown menu should be closed after click on item if locati
     assert.ok(!dropDown.option("opened"), "dropdown is closed");
     assert.equal(onClickActionStub.callCount, 1, "onClick was fired");
 });
+
+QUnit.module("adaptivity", {
+    beforeEach: function() {
+        this.element = $("#toolbar");
```

Comment:
Maybe add a `getItems` function here? You call `this.element.find("." + TOOLBAR_ITEM_CLASS)` in each test